src/app/api/hello/route.ts

```diff
@@ -2,6 +2,7 @@
 // Next.js Edge API Route Handlers: https://nextjs.org/docs/app/building-your-application/routing/router-handlers#edge-and-nodejs-runtimes
 
 import type { NextRequest } from 'next/server'
+import { Ai } from '@cloudflare/ai'
```

Comment:
This would work, except that Edge Runtime disallows `CompressionStream`, which `@cloudflare/ai` uses.

`https://github.com/vercel/edge-runtime/issues/750`